.github/workflows/deploy.yml

```diff
@@ -25,7 +29,7 @@ jobs:
       - name: Build application
         working-directory: sqs-tester
         run: |
-          ./gradlew build
+          ./gradlew build -x test
```

Comment:
this will build without running your tests - is that correct?

Reply:
@lkm0287 That was intentional. If I'm not mistaken the tests should run as part of the workflows/build.yml - the last line has 'run: ./gradlew build sonar --info' - so no need to do it twice?

Reply:
Ok thanks.